app/views/publishers/vacancies/job_applications/notes/index.html.slim

```diff
@@ -0,0 +1,29 @@
+- content_for :page_title_prefix, t(".page_title")
+
+= render "publishers/vacancies/job_applications/header"
+
+.govuk-grid-row
+  .govuk-grid-column-two-thirds
+    h2.govuk-heading-m = t(".heading")
+
+    .govuk-grid-row data-controller="show-hidden-content"
+      .govuk-grid-column-two-thirds
+        = govuk_button_to t(".add_note"), "#", secondary: true, data: { action: "click->show-hidden-content#show" }
+
+        = form_for @notes_form, id: "test", url: organisation_job_job_application_notes_path(vacancy.id, job_application), html: { data: { "show-hidden-content-target": "content" } } do |f|
```

Comment:
`id: "test"` ? why? rails sets a default id to forms why not using it?

Reply:
Ah that's leftover. Forgot to remove.